src/App.js

```diff
@@ -1,23 +1,88 @@
-import logo from './logo.svg';
+import { useState } from 'react';
 import './App.css';
 
 function App() {
+  const [products, setProducts] = useState([])
+  function GetAllProducts() {
+    const url = `http://localhost:3000/products`
+    fetch(url)
+      .then((res) => res.json())
+      .then((Data) => {
+        setProducts(Data)
+      })
+  }
```

Comment:
Need to be using a `useEffect` here.

Reply:
And use camel case ie. `Data` => `data` || `GetAllProducts` => `getAllProducts`.

The exception is for React components `App` which are capitalized.